@here/harp-mapview/lib/TileObjectsRenderer.ts

```diff
@@ -207,10 +221,26 @@ export class TileObjectRenderer {
      * @param object - The [[TileObject]] to process.
      * @returns `false` if the given [[TileObject]] should not be added to the scene.
      */
-    private processTileObjectFeatures(tile: Tile, object: TileObject): boolean {
-        const technique: IndexedTechnique = object.userData.technique;
+    private processTileObjectFeatures(
+        tile: Tile,
+        sotrageLevel: number,
```

Comment:
```suggestion
        storageLevel: number,
```

Reply:
actually, I didn't rename it because i'm not sure that the value is the storageLevel or the floored zoom level. I need to take a second look at what is going on there.